cmd/proxy.go

```diff
@@ -25,7 +30,26 @@ hostname and port where it listens.'`,
 func runProxyServer(cmd *cobra.Command, args []string) error {
 	glog.Infof("proxy server started at %s:%v", cfg.Hostname, cfg.Port)
 
-	ModelRegistryServiceAPIService := openapi.NewModelRegistryServiceAPIService()
+	mlmdAddr := fmt.Sprintf("%s:%d", proxyCfg.MLMDHostname, proxyCfg.MLMDPort)
+	glog.Infof("MLMD server %s", mlmdAddr)
+	conn, err := grpc.DialContext(
+		context.Background(),
+		mlmdAddr,
+		grpc.WithReturnConnectionError(),
+		grpc.WithBlock(),
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+	)
+	if err != nil {
+		log.Fatalf("Error dialing connection to mlmd server %s: %v", mlmdAddr, err)
+		return err
+	}
```

Comment:
We should add this defer to ensure cleaning up the connection resource, wdyt?
```suggestion
	if err != nil {
		log.Fatalf("Error dialing connection to mlmd server %s: %v", mlmdAddr, err)
		return err
	}
	defer conn.Close()
```